### ISSUE TITLE
got fatal error:  '../leptonica/allheaders.h' file not found when running make install

### ISSUE THREAD

**Issue by crokobit** [mauvilsa/tesseract-layout#1]
As title
got fatal error:  '../leptonica/allheaders.h' file not found when running make install
Is there anything I need to do before running `make file` ?

Many Thanks.

**Comment by mauvilsa** [comment]
On what platform are you trying to compile? What version of tesseract and leptónica do you have? Before the make I assume you ran cmake and leptonica was found without problem. So I am not sure why you are getting this error. If you give more details I may be able to help you.

I have only tested this on recent linux debian-based distributions and on mac os x with all prerequisites installed using macports.

**Comment by crokobit** [comment]
On macOS Sierra

install xcode for g++
brew install tesseract --all-languages
brew install leptonica

Here is the error after I ran `cmake -DCMAKE_INSTALL_PREFIX:PATH=$HOME ..` and `make install`.

```
Scanning dependencies of target tesseract-layout
[ 50%] Building CXX object CMakeFiles/tesseract-layout.dir/tesseract-layout.cc.o
/usr/local/Cellar/tesseract-layout/tesseract-layout.cc:12:10: fatal error: '../tesseract/baseapi.h' file not found
#include <../tesseract/baseapi.h>
         ^
1 error generated.
make[2]: *** [CMakeFiles/tesseract-layout.dir/tesseract-layout.cc.o] Error 1
make[1]: *** [CMakeFiles/tesseract-layout.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by mauvilsa** [comment]
I have committed a fix for the compilation when installing the prerequisites with brew. Do a pull and try it out to see if it works for you. In my system by default pkg-config is not able to find leptonica and tesseract so before the cmake I had to do
```
PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/Cellar/leptonica/1.74.4/lib/pkgconfig"
PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/Cellar/tesseract/3.05.01/lib/pkgconfig"
```

As a side note. Instead of this tool I recommend that you use the one at https://github.com/mauvilsa/tesseract-recognize that does exactly the same thing if called with the option --only-layout.
